wpilibj/src/main/java/edu/wpi/first/wpilibj/drive/RobotDriveBase.java

```diff
@@ -9,11 +9,12 @@
 
 import edu.wpi.first.wpilibj.MotorSafety;
 import edu.wpi.first.wpilibj.MotorSafetyHelper;
+import edu.wpi.first.wpilibj.SensorBase;
 
 /**
  * Common base class for drive platforms.
  */
-public abstract class RobotDriveBase implements MotorSafety {
+public abstract class RobotDriveBase extends SensorBase implements MotorSafety {
```

Comment:
????? A drive base isn't a sensor 👎

Shouldn't this be `extends SendableBase` anyway?